homeassistant/components/sensor/aftership.py

```diff
@@ -0,0 +1,130 @@
+"""
+Gives a sensor of all non-delivered packages recorded in AfterShip.
+
+For more details about this platform, please refer to the documentation at
+https://www.home-assistant.io/components/sensor.aftership/
+"""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import ATTR_ATTRIBUTION, CONF_API_KEY, CONF_NAME
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['pyaftership==0.1.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTRIBUTION = 'Information provided by AfterShip'
+
+DEFAULT_NAME = 'aftership'
+
+CONF_TITLE = 'title'
+CONF_SLUG = 'slug'
+
+CONF_TRACKING_NUMBER = 'tracking_number'
+
+ICON = 'mdi:package-variant-closed'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=30)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_API_KEY): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    })
+
+
+async def async_setup_platform(
+        hass, config, async_add_entities, discovery_info=None):
+    """Set up the AfterShip sensor platform."""
+    from pyaftership.tracker import Tracking
+
+    apikey = config[CONF_API_KEY]
+    name = config[CONF_NAME]
+
+    session = async_get_clientsession(hass)
+    aftership = Tracking(hass.loop, session, apikey)
+
+    await aftership.get_trackings()
+
+    if not aftership.meta or aftership.meta['code'] != 200:
+        _LOGGER.error("No tracking data found. Check API key is correct: %s",
+                      aftership.meta)
+        return
+
+    sensor = [AfterShipSensor(aftership, name)]
```

Comment:
This is a list so perhaps call it sensors, or move the list wrapping to the add entities call.

Reply:
fixed.